src/components/signup/signup.css

```diff
@@ -0,0 +1,44 @@
+
+
+    body {
+      background-color: #030301;
+      background-image: url('./peakpx\ 1.jpg');
+      background-repeat: no-repeat; 
+      background-size: cover; 
+      background-position: center
+    }
+
+    .signup-container, .signin-container {
```

Comment:
In the `signup.css` file you shouldn't define styles for the `signup-container`. Please check and ensure you remove styles and only keep those for the SignUp component should be here.